lib/elixir/lib/kernel.ex

```diff
@@ -3401,6 +3401,45 @@ defmodule Kernel do
   In the example above, a `sum/2` function is defined; this function receives
   two arguments and returns their sum.
 
+  ## Default arguments
+
+  `\\` is used to specify a default value for a parameter of a function. For
+  example:
```

Comment:
Then I don't really see the point. I'd rather have the Markdown be closed to what is rendered than having to keep these rules consistent.